src/main/java/com/epam/ta/reportportal/core/user/EditUserHandler.java

```diff
@@ -36,11 +35,11 @@
 	 *
 	 * @param username
 	 * @param editUserRQ
-	 * @param role
+	 * @param updater
 	 * @return
 	 * @throws ReportPortalException
 	 */
-	OperationCompletionRS editUser(String username, EditUserRQ editUserRQ, UserRole role);
+	OperationCompletionRS editUser(String username, EditUserRQ editUserRQ, ReportPortalUser updater);
```

Comment:
maybe 'editor' is a better word to wire method name and parameter?

Reply:
renamed